docs/sources/panels-visualizations/visualizations/trend/index.md

```diff
@@ -0,0 +1,31 @@
+---
+keywords:
+  - grafana
+  - graph panel
+  - trend panel
+  - documentation
+  - guide
+  - graph
+title: Trend
+weight: 90
+---
+
+# Trend
+
+The trend visualization type is the default and primary way to visualize data as a graph where x != time. It can render series as lines, points, or bars. It is versatile enough to display almost any sequential data.
```

Comment:
> The trend visualization type is the default and primary way to visualize data as a graph where x != time.

not sure about this. barcharts and xy don't have x time.

the intro can probably just provide concrete examples of datasets, such as:

> The trend visualization should be used for datasets that have sequential, numeric X that is _not_ time. Some examples are function graphs, rpm/torque curves, supply/demand relationships, elevation or heartrate plots along a route race route (with x as distance as duration from start).
>
> The trend panel supports all of the visual styles and options available in the [Time series panel]({{< relref "../time-series" >}}) with these exceptions:
>
> - No annotations or time regions
> - No shared crosshair
> - No multi-timezone x axis
> - No ability to change the dashboard time range via drag-selection